op-chain-ops/interopgen/recipe.go

```diff
@@ -79,8 +77,7 @@ func (r *InteropDevRecipe) Build(addrs devkeys.Addresses) (*WorldConfig, error)
 				DisputeGameFinalityDelaySeconds: big.NewInt(6),
 				MipsVersion:                     big.NewInt(1),
 			},
-			UseInterop:           true,
-			StandardVersionsToml: standard.VersionsMainnetData,
```

Comment:
Apologies if you might be repeating but why doesn't the superchain config rely on the `StandardVersionsToml` anymore? (and why it did in the past?)

Reply:
we used to read data from the toml to determine if we should route proxies to existing implementations. this is no longer necessary since we use deterministic deployments and figure this out inside the solidity scripts.

Reply:
@mslipper before op-deployer v0.2.0 we could deploy a full set of smart contracts on sepolia, including the `ProtocolVersions` proxies, whereas now we are getting a hard-coded `0x79ADD5713B383DAa0a138d3C4780C7A1804a8090` -- is this expected?